uk.co.compucorp.civicrm.hrleaveandabsences/api/v3/LeavePeriodEntitlement.php

```diff
@@ -44,3 +44,34 @@ function civicrm_api3_leave_period_entitlement_delete($params) {
 function civicrm_api3_leave_period_entitlement_get($params) {
   return _civicrm_api3_basic_get(_civicrm_api3_get_BAO(__FUNCTION__), $params);
 }
+
+/**
+ * LeavePeriod Entitlement.getremainder API
+ *
+ * @param $params
+ * @return array API result descriptor
+ * @throws InvalidArgumentException
+ */
+function civicrm_api3_leave_period_entitlement_getremainder($params){
+ if(empty($params['entitlement_id']) && (empty($params['contact_id']) || empty($params['period_id']))) {
```

Comment:
We need some tests to check that this validation is correct. Please check the tests in tests/phpunit/api/v3 to see how it can be done